### PR TITLE
Newsletter revival

### PR DESCRIPTION
**Pull request by jamesmunns**
Working on #121!

CC @eldruin @hargoniX @kalkyl, @therealprof, there are a couple to-dos left before we can publish:

1. ~We need to fill in all the placeholder links (like this: `[text](#)`) with actual links~
2. ~We need to count up the diffs for the Ecosystem crates at the bottom~ Done!

I think the best way to do this is to submit suggested changes, and we can merge them as they are ready. We can squash the final merge to keep the commit count down.

I'll have some more time later today to work on this, but feel free to contrib if you have time!

### CODE REVIEW

File: content/2020-08-24-newsletter-25.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 +++
-title = "The Embedded Working Group Newsletter - 24"
-date = 2020-02-29
+title = "The Embedded Working Group Newsletter - 25"
+date = 2020-08-24
```
Comment (jamesmunns) [inline content/2020-08-24-newsletter-25.md:3]: I've moved this to be the basis for next week's newsletter, so we don't lose out on the suggestions that have already been added.

**Comment by hargoniX** [comment]
Diff count up should be done with ^

**Comment by hargoniX** [comment]
Also added links for all the `(#)` I could find, I am not quite sure whether the link to the Rust compiler blog and the issue comment are "correct" here though, if someone has a better idea as to what we should link feel free to change it.

**Comment by jamesmunns** [comment]
Awesome, thanks @hargoniX! I'll give it a look tomorrow and get it ready for release! If anyone else has any changes/suggestions to propose, feel free! Otherwise I'll aim for a release around 14:00 CEST on Monday.

File: content/2020-08-17-newsletter-24.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+[the next newsletter]: https://github.com/rust-embedded/blog/edit/master/content/2020-08-24-newsletter-25.md
+
+## The Great Catch-Up!
+
+Here's a whirlwind overview of some of the things that have been released and changed over the past six months or so!
+We'll cover some of these updates in more detail over the coming weeks!
+
+### Renames
+
+* The RTFM framework has been renamed to RTIC - Real Time Interrupt-driven Concurrency!
```
Comment (eldruin) [inline content/2020-08-17-newsletter-24.md:46]:
```suggestion
* The RTFM framework has been renamed to [RTIC](https://rtic.rs) - Real Time Interrupt-driven Concurrency!
```

File: content/2020-08-17-newsletter-24.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+* [endian_codec](https://crates.io/crates/endian_codec)
+* [gdbstub](https://crates.io/crates/gdbstub)
+* [microfft](https://crates.io/crates/microfft)
+* [smbus-pec](https://crates.io/crates/smbus-pec)
+
+### New Possibilities
+
+* It is now possible run [async/await on embedded](https://blog.rust-lang.org/2020/06/04/Rust-1.44.0.html)!
+* Applications can now be compiled for [AVR in mainline Rust nightly builds](https://github.com/rust-lang/rust/issues/44052#issuecomment-663394401)!
+
```
Comment (eldruin) [inline content/2020-08-17-newsletter-24.md:140]:
```suggestion
```

File: content/2020-08-17-newsletter-24.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+### Tooling
+
+* the [probe-rs organization](https://probe.rs) has added a lot of wonderful new tooling, including [cargo-flash](https://crates.io/crates/cargo-flash) and [cargo-embed](https://crates.io/crates/cargo-embed)
+* [cargo-hf2](https://crates.io/crates/cargo-hf2) for "A small cargo subcommand to download cargo builds to Microsoft UF2 bootloaders via HID USB"
+* The [Knurling-rs](https://github.com/knurling-rs/meta) project for embedded tooling has kicked off by releasing [probe-run](https://github.com/knurling-rs/probe-run)
+
+### Bindings
+
+* [FreeRTOS-rust](https://crates.io/crates/freertos-rust) as RTOS bindings (not to be confused with [FreeRTOS.rs](https://crates.io/crates/freertos_rs) )
+
```
Comment (eldruin) [inline content/2020-08-17-newsletter-24.md:62]:
```suggestion
### New Possibilities

* It is now possible run [async/await on embedded](https://blog.rust-lang.org/2020/06/04/Rust-1.44.0.html)!
* Applications can now be compiled for [AVR in mainline Rust nightly builds](https://github.com/rust-lang/rust/issues/44052#issuecomment-663394401)!
```

**Comment by jamesmunns** [comment]
@rust-embedded/resources (also CC @eldruin @hargoniX @kalkyl), this should now be ready for release!

Please review and hit me with an r+!

**Comment by kalkyl** [comment]
Great job everyone! Sorry for being out of the loop the past couple of days, we just had a new baby! :)

**Comment by therealprof** [comment]
> Great job everyone! Sorry for being out of the loop the past couple of days, we just had a new baby! :)

Congrats! 🎉 🎈

**Pull request merged**